atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -186,6 +189,8 @@ def choose_cluster(self, platform):
         selected = min(clusters, key=lambda c: c.load)
         self.log.info('platform %s will use cluster %s',
                       platform, selected.cluster.name)
+        self.log.debug('platform %s will use cluster %s with the lowest load %s',
+                       platform, selected.cluster.name, selected)
```

Comment:
I don't really think we need this debug line if we have the get_cluster_info() one.